Formula/sheldon.rb

```diff
@@ -0,0 +1,20 @@
+class Sheldon < Formula
+  desc "Fast, configurable, shell plugin manager"
+  homepage "https://rossmacarthur.github.io/sheldon"
+  url "https://github.com/rossmacarthur/sheldon/archive/0.5.3.tar.gz"
+  sha256 "e471dd1ce97587b373313e2cd463a145db0f72573a22aae5a605f3b676258164"
+  license ["Apache-2.0", "MIT"]
```

Comment:
Multiple licenses aren't allowed until the next tagged `brew` release. Can you pick one and leave a comment with both licenses for now?